### PR TITLE
Added support for HAL_3_3.

### PR DESCRIPTION
**Pull request by ziquanyu**
Added changes for HAL3_3

### CODE REVIEW

**Comment by ziquanyu** [comment]
@emilwestergren  Can you review this when you have time? Thanks!

File: HAL/HAL3/CameraHALWrapper.cpp [hunk 1]
```diff
@@ -33,6 +35,7 @@ CameraHALWrapper::CameraHALWrapper(const char* cameraId) :
 	CameraMetadata meta;
 	meta = info.static_camera_characteristics;
 	ALOGI("Camera ID=%s", cameraId);
+#ifdef HAL_3_2 // HAL_3_3 ANDROID_LENS_OPTICAL_AXIS_ANGLE = ANDROID_LENS_POSE_ROTATION,
```
Comment (emilwestergren) [inline HAL/HAL3/CameraHALWrapper.cpp]: You can remove this comment.

File: HAL/HAL3/CameraHALWrapper.cpp [hunk 2]
```diff
@@ -40,6 +43,15 @@ CameraHALWrapper::CameraHALWrapper(const char* cameraId) :
 		this->cameraMeta.opticalAxisAngle[2] = opticalAxisAngle[2];
 		ALOGI("Optical axis angle: %f %f %f", opticalAxisAngle[0], opticalAxisAngle[1], opticalAxisAngle[2]);
 	}
+#elif HAL_3_3 // HAL_3_3 ANDROID_LENS_OPTICAL_AXIS_ANGLE = ANDROID_LENS_POSE_ROTATION,
```
Comment (emilwestergren) [inline HAL/HAL3/CameraHALWrapper.cpp]: This as well.

**Comment by emilwestergren** [comment]
Other than that it looks good. We will clean this up when we implement better HAL version macros.

**Comment by ziquanyu** [comment]
Updated.

**Pull request merged**